x-pack/plugins/enterprise_search/public/applications/shared/unsaved_changes_prompt/unsaved_changes_prompt.test.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+jest.mock('react-router-dom', () => ({
+  Prompt: () => null,
+}));
+import { Prompt } from 'react-router-dom';
```

Comment:
There should probably be a space here (I'll see if I can track down an eslint rule for this)

Reply:
eslint gets confused by `jest.mock` unfortunately. TBH I think it makes sense to put mocks right next to their imports for association also, so IMO this newline (or lack thereof) combination is +1 for me